management/server/http/setupkeys.go

```diff
@@ -190,16 +215,27 @@ func toResponseBody(key *server.SetupKey) *api.SetupKey {
 	} else {
 		state = "valid"
 	}
+
+	// the UpdatedAt field was introduced later, so there might be that some keys have a Zero value (e.g, null in the store file)
+	var updatedAt time.Time
+	if key.UpdatedAt.IsZero() {
```

Comment:
this check is already performed in the account management methods

Reply:
not really, but I will make sure it is only there